templates/kibana.yml.erb

```diff
@@ -1,5 +1,12 @@
 # This file is managed by Puppet, any changes will be overwritten
 
 <% scope['kibana4::config'].keys.sort.each do |key| -%>
+<% unless scope['kibana4::config'][key].is_a?(Array) -%>
```

Comment:
Good point :-) I'll look at inverting that into a normal if / else
On 2 Jun 2016 12:43 p.m., "Alex Harvey" <notifications@github.com> wrote:

> In templates/kibana.yml.erb
> <https://github.com/lesaux/puppet-kibana4/pull/65#discussion_r65518417>:
>
> > @@ -1,5 +1,12 @@
> >  # This file is managed by Puppet, any changes will be overwritten
> >
> >  <% scope['kibana4::config'].keys.sort.each do |key| -%>
> > +<% unless scope['kibana4::config'][key].is_a?(Array) -%>
>
> Use of unless...else is considered bad style e.g
> <https://signalvnoise.com/posts/2699-making-sense-with-rubys-unless>.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/lesaux/puppet-kibana4/pull/65/files/ec3e8c180d7321abe9741a1f6975f4baf939dc79#r65518417>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABQp3f1I8AW20iFSn3JeISiw3-cbMEB1ks5qHrPpgaJpZM4IpB0b>
> .
>